latex/Research_Strategy.tex

```diff
@@ -277,83 +277,71 @@
 Thus, we will likely need to turn to separating these issues to improve our handling of them one at a time. 
-For SAMPL7, then, our tentative plan is to measure pKa values for an extensive set of drug-like molecules in water and 
-%can we also do other solvent(s)?
-provide data specifically on pKa values, thereby separating the issues of predicting protonation state from those of transfer.
-{\color{red}[JDC: I don't believe the Sirius T3 is capable of p$K_a$ measurements for other solvents. I'm not even sure if that concept is meaningful since $K_a$ is defined with regard to free proton concentrations in water.}
+For SAMPL7, then, our tentative plan is to measure pKa values for an extensive set of drug-like molecules in water and provide data specifically on pKa values, thereby separating the issues of predicting protonation state from those of transfer and paving the way for SAMPL8.
 
 {\bf SAMPL8: pKa measurements and distribution coefficients.}
-In the next challenge, SAMPL8, we would re-combine the pKa and transfer issues in a way to maximize learning -- specifically, we will not only measure distribution coefficients but also measure pKa values for the same set of compounds, allowing participants to (a) predict distribution; (b) predict pKa; and (c) predict partitioning.
+In the next challenge, we will re-combine the pKa and transfer issues in a way to maximize learning -- specifically, we will not only measure distribution coefficients but also measure pKa values for the same set of compounds, allowing participants to (a) predict distribution; (b) predict pKa; and (c) predict partitioning.
 
 {\bf SAMPL9 and SAMPL10.}
-Several other avenues are of interest for future datasets as well, especially for SAMPL9 and SAMPL10.
-New computational techniques are targeting membrane permeability~\cite{lee_permeability_2016, comer_permeability_2014}, and this is experimentally accessible (see support letters from Pfizer and Merck), leading to potential interest in new datasets and challenges focused there.
-Alternatively, partition/distribution into other environments aside from cyclohexane or octanol may provide significant value, especially given dielectric constant issues posed by current force fields [ref].
-Solubility measurements may also be suitable for a late-stage SAMPL challenge such as SAMPL10, since solubility predictions are now beginning to become tractable~\cite{schnieders_structure_2012, park_absolute_2014, liu_using_2016} (with Schr\"{o}dinger also working on amorphous solubility prediction).
+We tentatively plan to focus on solubility prediction in SAMPL9, and membrane permeability in SAMPL10, because of new computational developments.
```

Comment:
Let's get rid of all of the waffle words and "tentatively" "hesitantly" "reservedly" language.

We can make it clear at the beginning that all plans are adaptable and able to exploit new opportunities, and just be concrete about what we envision with the information we have right now.